index.html

```diff
@@ -19,7 +19,7 @@
         shortName: "vc-data-model",
 
         // subtitle for the spec
-        subtitle: "Expressing verifiable information on the Web",
+        subtitle: "Expressing information with verifiable provenance",
```

Comment:
Suggesting this on behalf of @dhh1128. Can this achieve consensus?

```suggestion
        subtitle: "Expressing information with verifiable source",
```

Reply:
Ooof, this evokes a "What does that mean?" response from me... I don't see how it is any better than provenance.